sdk/digitaltwins/azure-digitaltwins-core/src/main/java/com/azure/digitaltwins/core/DigitalTwinsAsyncClient.java

```diff
@@ -1067,4 +1068,142 @@ public HttpPipeline getHttpPipeline() {
                 return Mono.just(new DigitalTwinsResponse<>(response.getRequest(), response.getStatusCode(), response.getHeaders(), null, twinHeaders));
             });
     }
+
+    /**
+     * Query digital twins.
+     * @param query The query string, in SQL-like syntax.
+     * @return A {@link PagedFlux} of application/json query result items.`
```

Comment:
nit: Is there an additional backtick here?

Reply:
ah yes :) You have a good eye!